content/blog/2018-03-20-rentrez-paper.md

```diff
@@ -0,0 +1,164 @@
+---
+slug: "rentrez paper" 
+title: A rentrez paper, and how to use the NCBI's new API keys 
+package_version: 1.2.1
+authors:
+    - name: David Winter
+    - url: https://david-winter.info
```

Comment:
@dwinter change the author block to:

```
authors:
    - name: David Winter
      url: https://david-winter.info
```

Reply:
the `https` version of your website seems to have a bad cert 😢

Reply:
Hmm, certificates sorted out but the https page does not render nicely. Will make sure to play with Jekyll and get it nice before the post goes up on the 20th